python/examples/core/harness.py

```diff
@@ -228,6 +229,44 @@ def _run_benchmark_n_iters(callback: Callable[[int], None], n_iters: int,
   return np.asarray([_pytimed(callback, *args) for _ in range(n_iters)])
 
 
+def test_argparser(benchmark_name: str,
+                   problem_sizes_list: Sequence[Sequence[int]],
```

Comment:
can we rename all default args with default_ ?

Reply:
Yes, I thought the names get lengthy but since these are the default values it makes sense to prefix